src/app/member/components/MemberDetials.test.tsx

```diff
@@ -0,0 +1,23 @@
+import { render, screen } from '@testing-library/react';
+import userEvent from "@testing-library/user-event";
+import { it, describe, expect} from 'vitest'; 
+import { MemberDetials } from './MemberDetials';
+
+describe('ActionsMenu Component', () => {
```

Comment:
can we change the name of this text  it should be `MemberDetail`

Reply:
@Mubashar1009  No, `MemberDetail` suggests a single piece of information, while this component displays multiple pieces of information. So, the current name is appropriate.

Reply:
fixed